pkg/kubeapiserver/options/plugins.go

```diff
@@ -160,6 +163,7 @@ func DefaultOffAdmissionPlugins() sets.Set[string] {
 		defaultingressclass.PluginName,          // DefaultIngressClass
 		podsecurity.PluginName,                  // PodSecurity
 		validatingadmissionpolicy.PluginName,    // ValidatingAdmissionPolicy, only active when feature gate ValidatingAdmissionPolicy is enabled
```

Comment:
this is GA now, right? So we can drop the comment.

Reply:
I'll leave it in a bit longer now that we have compatibility version merged and are tracking feature gates post-GA.